### PR TITLE
Fix typos in boarding missions.txt

### PR DESCRIPTION
**Pull request by AlexBassett**
**Bug fix**

## Summary
Adds spaces to reward phrases in boarding missions.txt to prevent word merging

### CODE REVIEW

File: data/human/boarding missions.txt [hunk 1]
```diff
@@ -150,9 +150,9 @@ phrase "assist merchant 3"
 	word
 		`captain `
 	word
-		`gives you`
-		`offers you`
-		`rewards you with`
+		`gives you `
+		`offers you `
+		`rewards you with `
```
Comment (alexrovw) [inline data/human/boarding missions.txt:155]: I don't believe these are necessary. At least, I didn't notice any issues with the spacing for this portion of the sentence.

Comment (alexrovw) [reply data/human/boarding missions.txt:155]: Looks fine without these changes:
image

Comment (AlexBassett) [reply data/human/boarding missions.txt:155]: after revisiting this section, I agree, I'll remove them from this push at the moment, and do some testing with the altered file.

**Review by alexrovw — APPROVED** [review]
Looks good now

**Pull request merged**